package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "task:1.1": "nodemon ./task1-1.js",
-    "task:1.2": "nodemon ./task1-2.js",
-    "task:1.3": "nodemon ./task1-3.js"
+    "lint": "eslint **.js",
+    "task:1.1": "nodemon ./src/task1-1.js",
+    "task:1.2": "nodemon ./src/task1-2.js",
+    "task:1.3": "nodemon ./lib/index.js"
```

Comment:
`"nodemon ./src/task1-3.js",`

Reply:
So, that was meant to be the compiled run after the babel command listed in the README. I've updated accordingly and brought it the one command.